pkg/agent/proxy/metrics/metrics.go

```diff
@@ -0,0 +1,128 @@
+package metrics
+
+import (
+	"sync"
+
+	kmetrics "k8s.io/component-base/metrics"
+	"k8s.io/component-base/metrics/legacyregistry"
+	"k8s.io/klog"
+)
+
+const (
+	metricNSAntrea = "antrea"
+	subsystemProxy = "proxy"
+)
+
+var (
+	once sync.Once
+
+	SyncProxyLatency = kmetrics.NewHistogram(
+		&kmetrics.HistogramOpts{
+			Namespace:   metricNSAntrea,
+			Subsystem:   subsystemProxy,
+			ConstLabels: map[string]string{"ip_family": "v4"},
+			Name:        "sync_proxy_rules_duration_seconds",
```

Comment:
I think that using latency here is confusing. Maybe use "duration" in the help text as well.

Reply:
Updated.